toolkit/tools/scheduler/schedulerutils/preparerequest.go

```diff
@@ -34,17 +34,24 @@ func ConvertNodesToRequests(pkgGraph *pkggraph.PkgGraph, graphMutex *sync.RWMute
 		}
 
 		req := &BuildRequest{
-			Node:           node,
-			PkgGraph:       pkgGraph,
-			AncillaryNodes: []*pkggraph.PkgNode{node},
-			IsDelta:        node.State == pkggraph.StateDelta,
+			Node:              node,
```

Comment:
This part might need a merge from `main`, since it changed quite drastically recently.

Reply:
Will do, will address feedback first, then give the merge a shot.